tools/unit_testing/run_tests.py

```diff
@@ -36,7 +36,7 @@ def parse_command_line(args):
     parser.add_argument("--build-dir", default=".",
                         help="""Directory where tests are built and run. Will be created if it does not exist."""
                         )
-    parser.add_argument("--build-type", default="CIME_DEBUG",
+    parser.add_argument("--build-type", default="CESM_DEBUG",
```

Comment:
The shr_infnan_mod test triggers off of this - so while it's not used in many places, it is used a bit.